tsrc/cli/main.py

```diff
@@ -123,6 +123,12 @@ def main(args=None):
     review_parser = workspace_subparser(subparsers, "review")
     review_parser.add_argument(
         "-t", "--target", dest="target_branch", default="master")
+    review_parser.add_argument(
+        "--all_pipeline", dest="all_pipeline", action="store_true", default=False)
```

Comment:
use dashes to separate words in options:  `--all-pipeline`.
don't worrry, argparse will replace dashes by undescores automatically :)

plus, all pipeline looks weird. `--whole-pipeline` ? `--full-pipeline` ?

Reply:
it's fixed in a later commit also

Reply:
--all is better?